bindings/nodejs/lib/types/models/native-token.ts

```diff
@@ -1,17 +1,20 @@
 // Copyright 2023 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+import { Transform } from 'class-transformer';
 import type { HexEncodedString } from '../utils/hex-encoding';
+import { hexToBigInt } from '../utils/hex-encoding';
 /**
  * Native token.
  */
-export interface INativeToken {
+export class INativeToken {
```

Comment:
Should we rename this type then in 2.0 (remove the `I` prefix)?

Reply:
yea, ill make at ticket